Makefile

```diff
@@ -45,11 +41,11 @@ generate-protobuf:
 	@ generate_protobuf_for() { \
 		local FILE="$$1"; \
 		local FILE_DIR="$$(dirname "$$GOPATH/$$FILE")"; \
-		echo "Generating protobuf file from $$FILE in $$FILE_DIR"; \
-		protoc -I "$$GOPATH/src/" -I '$(REPO_ROOT)/client/api' "$$FILE" --go_out=plugins="grpc:$(GOPATH)/src"; \
-	} ; \
-	export -f generate_protobuf_for; \
-	find '$(REPO_ROOT)' -name '*.proto' | sed -e "s|$(GOPATH)/src/||g" | xargs -n1 $(SHELL) -c 'generate_protobuf_for "$$0"'
+                echo "Generating protobuf file from $$FILE in $$FILE_DIR"; \
+                protoc -I "$$GOPATH/src/" -I '$(REPO_ROOT)/client/api' "$$FILE" --go_out=plugins="grpc:$(GOPATH)/src"; \
```

Comment:
Better use `$(GOPATH)` consistently, i.e. replace `$$GOPATH`.

Reply:
fixed it.